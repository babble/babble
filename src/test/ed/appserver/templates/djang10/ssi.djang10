{% ssi /local/djang10-if.djang10 parsed %}
{% ssi /local/djang10-if.djang10 %}
