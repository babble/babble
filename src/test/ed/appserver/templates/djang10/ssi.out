Should see me1

Should see me 2

Should see me 3


{% if foo %}
Should see me1
{% else %}
Should Not See Me1
{% endif %}

{% if notFoo %}
Should not see me2
{% else %}
Should see me 2
{% endif %}

{% if foo and not notFoo %}
Should see me 3
{% endif %}